gobblin-core/src/main/java/org/apache/gobblin/converter/parquet/JsonIntermediateToParquetConverter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gobblin.converter.parquet;
+
+import org.apache.gobblin.configuration.WorkUnitState;
+import org.apache.gobblin.converter.Converter;
+import org.apache.gobblin.converter.DataConversionException;
+import org.apache.gobblin.converter.SchemaConversionException;
+import org.apache.gobblin.converter.SingleRecordIterable;
+import org.apache.gobblin.converter.parquet.JsonElementConversionFactory.RecordConverter;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+
+import parquet.schema.MessageType;
+
+import static org.apache.gobblin.converter.parquet.JsonElementConversionFactory.RecordConverter.RecordType.ROOT;
+
+
+/**
+ * A converter to Convert JsonIntermediate to Parquet
+ * @author tilakpatidar
+ */
+public class JsonIntermediateToParquetConverter extends Converter<JsonArray, MessageType, JsonObject, ParquetGroup> {
+  private RecordConverter recordConverter;
+
+  @Override
+  public MessageType convertSchema(JsonArray inputSchema, WorkUnitState workUnit)
+      throws SchemaConversionException {
+    String fieldName = workUnit.getExtract().getTable();
+    JsonObject temp = new JsonObject();
```

Comment:
Please provide a more meaningful name, as it's passed to construct a `RecordConverter`

Reply:
👍